tetris/index.js

```diff
@@ -0,0 +1,121 @@
+var express = require('express');
+var app = express();
+var http = require('http').Server(app);
+var io = require('socket.io')(http);
+var path = require('path');
+var util = require('util');
+
+var users = require(__dirname + '/js/userContainer.js')();
+var players = require(__dirname + '/js/playerContainer.js')();
```

Comment:
`./js/playerContainer.js` after you have moved things down out of the tetris folder.